homeassistant/components/tuya/switch.py

```diff
@@ -430,6 +430,14 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
+            name="Switch",
```

Comment:
Can't we give this switch a translated name, preferable better than "Switch"?

Reply:
What do you suggest? Value?

Reply:
Remove `name` and add a translation key that matches the "Switch" value. E.g., `translation_key="switch"`.